core/src/test/java/org/apache/carbondata/core/scan/filter/FilterUtilTest.java

```diff
@@ -223,17 +223,6 @@ public void testCheckIfRightExpressionRequireEvaluationWithExpressionIsInstanceO
             DataTypes.STRING) instanceof ColumnFilterInfo);
   }
 
-  @Test public void testCreateBitSetGroupWithDefaultValue() {
```

Comment:
Instead of removing, change the test case to use the new method

Reply:
done